docker/production/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.4'
+
+volumes:
+  public:
+  db:
+  caddy_data:
+
+services:
+  db:
+    image: postgres
+    restart: always
+    volumes:
+      - db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: mat
+      POSTGRES_DB: mutualaidtompkins
```

Comment:
We probably want installation specific values to not be hardcoded in the main repo, yeah?
Would be fine with extracting them in a later PR though.
There are some further down in this file as well.